website/case-studies/implementation-of-flint-modules.md

```diff
@@ -0,0 +1,407 @@
+
+|![Diagram depicting the high-level design of FLINT.](../static/img/flint-design-module-case-study.png) |
+|:--:|
+| Diagram depicting the high-level design of FLINT. |
+
+## Empirical Forest Growth Module
+
+The empirical Forest growth module includes all the operations required to simulate the biomass accumulation. Specifically, it simulates the growth in carbon stock within a forest and its components through time. It is a standard module used in FLINT and is used within Chapman Richards Module. **Biomass** refers to the mass of living organisms, including plants, animals and microorganisms. **Biomass accumulation** is the net change in standing biomass from one time to another (g/m^2).
+
+It runs at the **Monthly Step Interval**. It is based on the yield tables that present the **monthly growth increments** for a forest type. This data comes from research and experimentation and is called **empirical data**, thus giving the name of this module as empirical forest growth module. Yield Tables only include the **aboveground biomass**.
+
+### Inputs:
+
+-   Time Step (Monthly)
+-   Species
+-   Age
+-   Region
+-   Forest type
+-   Pool 1: Stemwood
+-   Pool 2: Bark
+-   Pool 3: Leaves
+-   Pool 4: Coarse Root
+-   Pool 5: Fine Roots
+-   Rainfall in that region
+-   Minimum and Maximum Temperature
+-   [Vapour Pressure Deficit](https://en.wikipedia.org/wiki/Vapour-pressure_deficit)
+-   Soil Water Holding Capacity
+
+### Outputs:
+
+-   Calculated aboveground and belowground Biomass.
+
+## Hybrid Forest Growth Module
+
+The Hybrid Forest Growth Module calculates the biomass above and below the ground. These modules use physiological responses to environmental conditions to derive relationships that can be used to predict tree growth. These modules contain equations or series of equations to calculate biomass.
+
+Like in `forestgrowthmodule.cpp`, the equation used is:
+
+```
+double ForestGrowthModule::calculate_above_ground_biomass(double age, double max, double k, double m) {
+    return max * pow(1 - exp(-k * age), 1 / (1 - m));
+}
+```
+
+The Hybrid Forest Growth module represents a 3-PG model (Physiological Principles in Predicting Growth). 3-PG models require finer detailed data than is typically represented in empirical growth models. 3-PG models are better than the empirical models as they can be used with any landscape as long as its variables are present. The GCBM modules contain one example of an empirical growth module.
+
+## Wofost Crop Growth Module
+
+WOFOST (WOrld FOod STudies) is a mechanistic module that simulates crop growth on the basis of photosynthesis, respiration and the prevailing environmental conditions. It is similar to the hybrid forest growth module. It is not included in the FLINT and is specially designed for **KENYA**.
```

Comment:
I don't think the WOFOST model is specific to Kenya, but rather it has been implemented as part of SLEEK (which is Kenya's FLINT implementation). Because of it's limited scope for application I would move it to near the bottom of this section.